conf/messages.cy

```diff
@@ -490,37 +490,37 @@ global.page.not.found.error.contact.link.text=cysylltwch â Gwasanaeth Cwsmeriai
 global.page.not.found.error.contact.link=https://www.gov.uk/government/organisations/hm-revenue-customs/contact/welsh-language-helplines
 
 
-label.access_your_self_assessment=Access your Self Assessment
+label.access_your_self_assessment=Cael at eich cyfrif Hunanasesiad
 
-title.selfAssessment.viewPayments.h1=Self Assessment payments for the last 60 days
-label.selfAssessment.noPaymentsIn60=There have been no payments in the last 60 days
-label.selfAssessment.balanceUpdateAdvisory=It can take up to 7 days to update your balance after each payment.
-label.selfAssessment.paymentsTable.date=Date
-label.selfAssessment.paymentsTable.reference=Reference number
-label.selfAssessment.paymentsTable.amount=Amount
+title.selfAssessment.viewPayments.h1=Taliadau Hunanasesiad ar gyfer y 60 diwrnod diwethaf
+label.selfAssessment.noPaymentsIn60=Ni fu unrhyw daliadau yn ystod y 60 diwrnod diwethaf
+label.selfAssessment.balanceUpdateAdvisory=Gall gymryd hyd at 7 diwrnod i ddiweddaru’ch balans ar ôl pob taliad.
+label.selfAssessment.paymentsTable.date=Dyddiad
+label.selfAssessment.paymentsTable.reference=Cyfeirnod
+label.selfAssessment.paymentsTable.amount=Swm
```

Comment:
These can be removed from the English and Welsh messages files, the page they were used on no longer exists

Reply:
Agreed
Deleted the unused messages